docs/examples/frameworks/jax/pax-basic_example.ipynb

```diff
@@ -0,0 +1,1482 @@
+{
```

Comment:
<p>Not having read the whole notebook yet, I don, know how to interpret this sentence:</p><p> </p><p><em>We will create a simple wrapper that uses </em><code><em>DALIGenericIterator</em></code><em> for JAX to connect DALI and PAX </em><code><em>Experiment</em></code></p><p> </p><p>Is it: </p><p><em>We will create a simple wrapper that uses </em><code><em>DALIGenericIterator</em></code><em> for JAX to connect (DALI and PAX </em><code><em>Experiment)</em></code></p><p><em>We will create (a simple wrapper that uses </em><code><em>DALIGenericIterator</em></code><em> for JAX to connect DALI) and (PAX </em><code><em>Experiment)</em></code></p><p> </p><p>In case of the former, I'd recommend rephrasing as:</p><p> </p><p>We will create a simple wrapper that uses <code>DALIGenericIterator</code> for JAX <s>to</s> <strong>that</strong> connect<strong>s</strong> DALI <s>and</s> <strong>to</strong> PAX <code>Experiment</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/5060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>